server/chat-plugins/mafia.ts

```diff
@@ -3433,8 +3442,8 @@ export const commands: ChatCommands = {
 			if (!logs.leaderboard[month]) logs.leaderboard[month] = {};
 
 			let toGiveTo = [];
-			let buf = `${points} point${Chat.plural(points, 's were', ' was')} awarded to: `;
-			if (cmd === 'winfaction') {
+			let buf = `${points < 0 ? points * -1 : points} point${Chat.plural(points, 's were', ' was')} ${points <= 0 ? 'taken from ' : 'awarded to '} `;
```

Comment:
I feel like an intermediate variable along the lines of `const isUnwin = cmd.startsWith('unwin');` would simplify a lot of this code.